plat/kvm/arm/setup.c

```diff
@@ -231,5 +194,6 @@ void __no_pauth _ukplat_entry(void)
 	 */
 	uk_pr_info("Switch from bootstrap stack to stack @%p\n", bstack);
 
-	lcpu_arch_jump_to(bstack, _ukplat_entry2);
+	lcpu_arch_jump_to(bstack, uk_boot_entry);
```

Comment:
I guess all calls to `uk_boot_entry` imply adding `lib/ukboot` as dependency (`select`) to the corresponding platform's `Config.uk`.